doc/radosgw/notifications.rst

```diff
@@ -144,6 +148,20 @@ Request parameters:
   - "none": message is considered "delivered" if sent to broker
   - "broker": message is considered "delivered" if acked by broker (default)
 
+- AWS Endpoint
+
+ - push-endpoint: ``aws://sns.amazonaws.com`` for AWS SNS and ``aws://lambda.amazonaws.com`` for AWS lambda.
+ - use-ssl: if "true" then secure connection will be used for connecting with the broker
+ - AccessKey: Access Key for access to AWS services
+ - SecretKey: Access secret for access to AWS services
+ - AwsArn: ARN of the resource to push notification to.
```

Comment:
please explain what are the valid ARNs: "sns" and "lambda"

Reply:
@prateek2211 please provide a sample of valid sns and lambda ARNs we expect

Reply:
done